dockerfiles/init/modules/openshift/files/scripts/che-config-map

```diff
@@ -0,0 +1,34 @@
+# To put some extra CHE server configurations just add here CHE ENV vars following format:
+# KEY: "value"
+CHE_API: ${HTTP_PROTOCOL}://che-${OPENSHIFT_NAMESPACE_URL}/api
+CHE_DEBUG_SERVER: ${CHE_DEBUG_SERVER}
+CHE_HOST: che-${OPENSHIFT_NAMESPACE_URL}
+CHE_INFRASTRUCTURE_ACTIVE: openshift
+CHE_INFRA_OPENSHIFT_BOOTSTRAPPER_BINARY__URL: ${HTTP_PROTOCOL}://che-${OPENSHIFT_NAMESPACE_URL}/agent-binaries/linux_amd64/bootstrapper/bootstrapper
+CHE_INFRA_OPENSHIFT_MACHINE__START__TIMEOUT__MIN: "5"
+CHE_INFRA_OPENSHIFT_MASTER__URL: ""
+CHE_INFRA_OPENSHIFT_OAUTH__TOKEN: ${CHE_INFRA_OPENSHIFT_OAUTH__TOKEN}
+CHE_INFRA_OPENSHIFT_PASSWORD: ${CHE_INFRA_OPENSHIFT_PASSWORD}
+CHE_INFRA_OPENSHIFT_PROJECT: ${CHE_INFRA_OPENSHIFT_PROJECT}
+CHE_INFRA_OPENSHIFT_PVC_STRATEGY: ${CHE_INFRA_OPENSHIFT_PVC_STRATEGY}
+CHE_INFRA_OPENSHIFT_TLS__ENABLED: ${ENABLE_SSL}
+CHE_INFRA_OPENSHIFT_TRUST__CERTS: "false"
+CHE_INFRA_OPENSHIFT_USERNAME: ${CHE_INFRA_OPENSHIFT_USERNAME}
+CHE_KEYCLOAK_AUTH__SERVER__URL: ${CHE_KEYCLOAK_AUTH__SERVER__URL}
+CHE_KEYCLOAK_CLIENT__ID: ${CHE_KEYCLOAK_CLIENT__ID}
+CHE_KEYCLOAK_GITHUB_ENDPOINT: ${KEYCLOAK_GITHUB_ENDPOINT}
+CHE_KEYCLOAK_OSO_ENDPOINT: ${CHE_KEYCLOAK_OSO_ENDPOINT}
+CHE_KEYCLOAK_REALM: ${CHE_KEYCLOAK_REALM}
+CHE_LOCAL_CONF_DIR: /etc/conf
+CHE_LOGS_DIR: /data/logs
+CHE_LOG_LEVEL: ${CHE_LOG_LEVEL}
+CHE_MULTIUSER: ${CHE_MULTIUSER}
+CHE_OAUTH_GITHUB_CLIENTID:
```

Comment:
Should it be empty value here or `${CHE_OAUTH_GITHUB_CLIENTID}` env var?

Reply:
I thought by default yes, it should be empty and I probably have to remove it from deploy script. as this file is supposed to be edited to set this configuration. But if we will set value to `${CHE_OAUTH_GITHUB_CLIENTID}` that would respect global ENVs and will pick it if it is set. So I think we could do same for all ENVs that are empty by default.